src/routes/(pages)/blog/posts/nais-console/+page.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "A Nais Console"
+description: "How Nais Console evolved into a unified developer portal, its product philosophy, and the technology stack behind it."
+date: 2025-05-30T11:37:43+02:00
+draft: false
+author: Hans Kristian Flaatten
+tags:
+  - developer-experience
+  - platform
+  - svelte
+  - kubernetes
+  - graphql
+  - product-development
+  - nais
+language: en
+---
+
+Nais has always been a platform for technically skilled developers building applications. The vision has been to let developers stay in their flow, using the tools they prefer, while minimizing the need to interact with the platform itself. In other words, the goal has never been about maximizing page views or time spent on the platform.
+
+Our primary interfaces are documentation and configuration as code (`nais.yaml`), where developers define how their applications should run on Nais. This integrates seamlessly with their workflows using GitHub Actions. Additionally, we’ve provided tools like Prometheus and Grafana for dashboards, Kibana for logs, and kubectl for troubleshooting. These tools have served us well for a long time.
+
+Over time, we’ve built specialized interfaces for managing different parts of the platform, such as Nais Deploy for API keys and deployment status, Nais Teams for team registration and access management, Metabase for cost insights, and Dependency Track for vulnerability tracking.
+
+![Nais Deploy Screenshot](./images/nais-deploy.png)
+
+<small>Screenshot of Nais Deploy user interface</small>
+
+However, as the platform has grown to support hundreds of teams and nearly 2,000 applications, we’ve seen a diverse range of users with varying backgrounds and needs. There’s also an expectation for a more unified and streamlined user experience compared to when we started building Nais nearly a decade ago.
+
+![Nais Console Timeline](./images/nais-timeline.svg)
```

Comment:
Kunne vi fått inn en pause-knapp for animasjoner? Personlig syns det er distraherende når jeg skal lese noe (kanskje ironisk med hvor mye jeg bruker det i Slack).

Reply:
Hvordan pauser man en gif? 🤔

Reply:
Mange bruker VP8/VP9 i stedetfor gif for å redusere størrelse og støtte f.eks. pause (siden det er video). Men trenger da å bygge fallback etc